.gitpod.Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM gitpod/workspace-full
```

Comment:
Where did the rest of the .gitpod.Dockerfile go ?

https://github.com/united-manufacturing-hub/ManagementConsole/blob/staging/.gitpod.Dockerfile

Reply:
There was no `.gitpod.Dockerfile` in this repo. Does this repo also need the rest of configuration like https://github.com/united-manufacturing-hub/ManagementConsole/blob/staging/.gitpod.Dockerfile?

Reply:
nvm, I added the rest of configuration.